go/tasks/plugins/hive/executor.go

```diff
@@ -94,7 +96,7 @@ func (q QuboleHiveExecutor) Finalize(ctx context.Context, tCtx core.TaskExecutio
 		return errors.Wrapf(errors.CorruptedPluginState, err, "Failed to unmarshal custom state in Finalize")
 	}
 
-	return Finalize(ctx, tCtx, q.resourceNamespace, incomingState)
+	return Finalize(ctx, tCtx, q.GetResourceNamespace(), incomingState)
```

Comment:
One more question - what happens if a user requests a resource from say, a new cluster in Qubole, that has not yet been added to the Config object?  It'll fail right?

Reply:
Good point. I think I'll panic. Let me fix it